### PR TITLE
Created functional InitalValues menu

### PR DESCRIPTION
**Pull request by WyattHolliday**
Here are values, their possible range and increment rate
Parameter                                        Min  Max  Inital Stepsize
CellSpawner::numCells	                0      100   10     1
CellSpawner::resourceProportion    0      1       0.75  0.05
CellEnvironment::nNutrientZones   0      50     3      1
NutrientZone::nutrientMaximum    0      10k   1000 10
NutrientZone::feedingRate             0.01 100  0.1    0.01
NutrientZone::regenerationRate    0      100  1       0.1
CellState::lifespan                           1      1000 60    1
CellState::growthNutrientCost        0      10k   1000 50
CellState::growthEnergyCost          0      10k   1000 50
And of course NutrientZone::deleteOnEmpty is simply true or false
and lots of cell structure parameters

The only thing I was thinking about changing was 10,000 for growthNutrientCost and growthEnergyCost. Perhaps to a lower number for the user and then scaling up again behind the scenes

Also, I found that you can use the scroll wheel to change the values and that seems much easier than clicking

### CODE REVIEW

File: src/cell_structure.cpp [hunk 1]
```diff
@@ -59,8 +59,9 @@ float CellStructure::getScale() const { return _scale; }
 void CellStructure::setSprite(AnimatedSprite2D *sprite) { _sprite = sprite; }
 AnimatedSprite2D *CellStructure::getSprite() { return _sprite; }
 Size2 CellStructure::getSpriteSize() {
-	if (_sprite)
+	try {
 		return _sprite->get_sprite_frames()->get_frame_texture("activate", 0)->get_size();
-
-	return Size2();
+	} catch (...) {
```
Comment (Alex-x90) [inline src/cell_structure.cpp:64]: Because exception handling is disabled, this is causing all the builds (except windows msvc) to fail.

**Comment by Alex-x90** [comment]
Nitpick (feel free to ignore): A lot of the code in the misc. _ready() functions is repeated and feels like it could potentially be moved to a function or perhaps the base class they're all based on.

File: src/cell_structure.cpp (hunk 1 above)
Comment (MylesScholz) [reply src/cell_structure.cpp]: Fixed!

**Review by Alex-x90 — APPROVED** [review]
Looks good to me!

**Pull request merged**